controllers/registration.js

```diff
@@ -433,6 +433,7 @@ router.get(['/registration/:classOrSchoolId', '/registration/:classOrSchoolId/:s
 			invalid,
 			secure,
 			correctID,
+			cloudConsentNecessary: res.locals.theme.cloud_consent_necessary
```

Comment:
this should be now `res.locals.theme.consent_necessary`, right?

Reply:
Yup, you're right :D